mmdet3d/core/evaluation/semantickitti_eval.py

```diff
@@ -0,0 +1,232 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+
+
+class SemanticKITTIEval(object):
```

Comment:
Does this class come from the official devkit? If not, please add corresponding docstrings.